src/main/jhipster/src/main/webapp/app/germplasm-list/list.component.html

```diff
@@ -69,7 +68,7 @@
 						data-test="editListMetadataButton">
 					<i [ngClass]="'fa fa-edit'"></i>
 				</button>
-				<button class="btn btn-light" (click)="toggleListStatus()" [disabled]="!(principal.hasAnyAuthorityDirect(ADMIN_PERMISSIONS) || user?.id === germplasmList?.ownerId)" data-test="lockListButton">
+				<button class="btn btn-light" (click)="toggleListStatus()" [disabled]="!principal.hasAnyAuthorityDirect(LOCK_UNLOCK_PERMISSIONS) && user?.id !== germplasmList?.ownerId" data-test="lockListButton">
```

Comment:
sounds like it should be possible to keep the old format still: `!(has authority.. || isowner..)`, maybe a little more easy to follow. See if it's possible, but otherwise this should be fine too